x-pack/plugins/telemetry_collection_xpack/README.md

```diff
@@ -0,0 +1,9 @@
+# telemetry_collection_xpack
+
+> Add the telemetry collector with the xpack information
+
+---
+
+## Development
+
+See the [kibana contributing guide](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md) for instructions setting up your development environment.
```

Comment:
nit: 'See the [kibana contributing guide](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md) for instructions on how to set up your development environment.'

Reply:
ha! good call! this is actually an auto-generated message by the `node scripts/generate_plugin.js` script :)